### ISSUE TITLE
GameMode failing tests after default inital installation

### ISSUE THREAD

**Issue by AlexSwartz** [FeralInteractive/gamemode#126]
**Describe the bug**
Followed default installation instructions, run gamemoded -t and tests are failed.
Placing gamemoderun %command% in launch options for any steam game causes the game to open and instantly close.

**To Reproduce**
Steps used to reproduce the behavior:
```
apt install meson libsystemd-dev pkg-config ninja-build git dbus-user-session
git clone https://github.com/FeralInteractive/gamemode.git
cd gamemode
git checkout 1.3 # omit to build the master branch
./bootstrap.sh

gamemoded -t
```

**Expected behavior**
Pass all the tests

**System Info (please complete the following information):**
 - OS and version: Ubuntu 18.04.2

 - GameMode Version: 1.3
- Processor: Intel Core m3-6Y30 CPU @ 0.90GHz × 4
- Graphics: Intel HD Graphics 515 (Skylake GT2)
- GNOME: 3.28.2

**Additional context**

```
alex@alex-laptop:~$ gamemoded -t
: Loading config
: Running tests

:: Basic client tests
:: Passed

:: Dual client tests
gamemode request succeeded and is active
Quitting by request...
:: Passed

:: Gamemoderun and reaper thread tests
ERROR: failed to launch gamemoderun with execl: Permission denied
...Waiting for child to quit...
...Waiting for reaper thread (reaper_frequency set to 5 seconds)...
ERROR: gamemode failed to deactivate when requested (expected 0)!
:: Supervisor tests
ERROR: gamemode_query_status after end request gave unexpected value 1, (expected 0)!
ERROR: :: Failed!
: Client tests failed, skipping feature tests
: Tests Failed!
```

**Comment by aejsmith** [comment]
There was a bug affecting older Meson versions (including the one in Ubuntu 18.04) that caused gamemoderun to be installed with incorrect permissions - see #115. This is fixed in 1.3.1, could you try checking out the 1.3.1 tag and installing from there? You may need to remove the old build folder before rebuilding.

**Comment by AlexSwartz** [comment]
1.3.1 is working, passing all the tests now. Thanks!
